homeassistant/components/upnp/sensor.py

```diff
@@ -65,6 +69,7 @@
         native_unit_of_measurement=TIME_SECONDS,
         entity_registry_enabled_default=False,
         format="d",
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
Is not a measurement. As a matter of fact, this sensor is actually against our guidelines, to begin with.

I suggest on removing this state class.

Reply:
I plan on deprecating then removing the sensors, but I need to find a proper alternative first.